metadata-ingestion/src/datahub/ingestion/source/qlik_sense/qlik_sense.py

```diff
@@ -252,17 +257,23 @@ def _gen_charts_workunit(self, charts: List[Chart]) -> Iterable[MetadataWorkUnit
 
             yield self._gen_entity_status_aspect(chart_urn)
 
+            custom_properties = {
+                "Dimension": str(chart.qDimension),
+                "Measure": str(chart.qMeasure),
+            }
```

Comment:
this info should not go in custom properties - we have actual models for these things, and should be able to see "Fields" in the UI. See this screenshot from Looker as an example

<img width="995" alt="image" src="https://github.com/datahub-project/datahub/assets/12566801/4a333f13-a799-44a7-b270-0b226d36be04">

Reply:
To generare above fields in charts we require parent urn (Dataset urn) of field which we are not able to generate here as of now.